### ISSUE TITLE
Ability to define Phab conduit credentials per job?

### ISSUE THREAD

**Issue by kosfar** [uber/phabricator-jenkins-plugin#168]
Hi,
if I define more than 1 Phab conduit credentials in Jenkins, it seems that I cannot select which one a single job will use. If "Default Phabricator Credentials" is none, then build jobs fail and if I choose one from the list, then jobs that need to talk to other Phab instances fail. 

Do I miss anything? Nice job btw.

Regards,
Kostis

**Comment by sectioneight** [comment]
Nope, this is probably just a bug/limitation in the implementation. In every set up I've seen, there's been a 1:1 mapping between Jenkins and Phab instances, so I went "full YAGNI" and didn't write anything to handle multi-phab-per-jenkins setups.

In your case, are you talking to different phabricator installs, or is it that you want to use different users on the same install?

**Comment by kosfar** [comment]
Yes, my initial intention was to configure credentials for repos across two phab instances and have jobs that will be based on both of them. I don't know if you want to consider it as a feature request for the future but right now I realize I need to go 1:1 between Jenkins and Phab instances. I will close this if you don't mind.

**Comment by sectioneight** [comment]
I think it's a valid feature request, I'm just not going to have time to
implement it any time soon.

I think we should leave the issue open, and maybe you or someone else will
have time to implement it.
